netebpfext/net_ebpf_ext.c

```diff
@@ -227,6 +227,42 @@ static net_ebpf_ext_wfp_callout_state_t _net_ebpf_ext_wfp_callout_states[] = {
 // WFP globals
 static HANDLE _fwp_engine_handle;
 
+// We wait on this rundown during driver unload to ensure that we get 'filter deleted'
+// notifications from WFP for all our filters.  We acquire this rundown when we associate the
```

Comment:
nit: would it be simpler if we mention that this rundown is acquired when `filter_context` is created, and released when the context is deleted?

Reply:
fixed.